src/components/NavBarDrawer.css

```diff
@@ -1,3 +1,9 @@
 img {
   height: 1.2em;
 }
+button {
```

Comment:
Please do not make changes targeting entire HTML tags.
This could have adverse effects elsewhere.

Also, I don't think we're depending on this anywhere, as buttons are rarely `display:flex` in our code. I could be wrong though